mbuild/tests/test_compound.py

```diff
@@ -393,10 +407,10 @@ def test_intermol_conversion2(self, ethane, h2o):
     def test_parmed_conversion(self, ethane, h2o):
         compound = mb.Compound([ethane, h2o])
 
-        structure = compound.to_parmed()
+        structure = compound.to_parmed(box=compound.boundingbox)
```

Comment:
That is if they call `save()`. If they just call `to_parmed`, that is not currently happening. 

Unsure how we would want to handle this actually. Since we were wanting the padding to be handled by the saver. I guess I could check if the box passed in is `None`, if so, pad and move on. That would handle use cases when just calling this function. Since `save()` does the padding before passing that box to `to_parmed()`